centreon/tests/e2e/cypress/e2e/Dashboards/06-dashboard-sharing/index.ts

```diff
@@ -0,0 +1,797 @@
+  cy.getByLabel({
+    label: 'view',
+    tag: 'button'
+  })
+    .contains(dashboards.fromDashboardAdministratorUser.name)
+    .click();
+  cy.getByLabel({ label: 'share', tag: 'button' }).click();
+});
+
+Then('the user is redirected to the sharing list of the dashboard', () => {
+  cy.contains('Manage access rights').should('be.visible');
+  cy.get('*[class^="MuiList-root"]').eq(1).should('exist');
+});
+
+Then('the creator of the dashboard is listed as its sole editor', () => {
+  cy.get('*[class^="MuiList-root"]')
+    .eq(1)
+    .children()
+    .its('length')
+    .should('eq', 1);
+  cy.get('*[class^="MuiList-root"]')
+    .eq(1)
+    .children()
+    .eq(0)
+    .should('contain', `${dashboardAdministratorUser.login}`);
+  cy.getByTestId({ testId: 'role-input' })
+    .eq(1)
+    .should('contain.text', 'editor');
+});
+
+Given('a non-admin user who has update rights on a dashboard', () => {
+  cy.loginByTypeOfUser({
+    jsonName: dashboardCreatorUser.login,
+    loginViaApi: false
+  });
+  cy.visit(`${Cypress.config().baseUrl}/centreon/home/dashboards`);
+});
+
+When('the editor user sets another user as a viewer on the dashboard', () => {
+  cy.getByLabel({
+    label: 'view',
+    tag: 'button'
+  })
+    .contains(dashboards.fromDashboardCreatorUser.name)
+    .click();
+  cy.getByLabel({ label: 'share', tag: 'button' }).click();
+  cy.getByLabel({ label: 'Open', tag: 'button' }).click();
+  cy.contains(dashboardViewerUser.login).click();
+  cy.getByTestId({ testId: 'add' }).should('be.enabled');
+  cy.getByTestId({ testId: 'role-input' }).eq(0).click();
+  cy.get('[role="listbox"]').contains('viewer').click();
+  cy.getByTestId({ testId: 'add' }).click();
+
+  cy.get('*[class^="MuiList-root"]')
+    .eq(1)
+    .children()
+    .its('length')
+    .should('eq', 2);
+  cy.get('*[class^="MuiList-root"]')
+    .eq(1)
+    .children()
+    .eq(0)
+    .should('contain', `${dashboardViewerUser.login}`);
+
+  cy.get('[data-state="added"]').should('exist');
+  cy.getByLabel({ label: 'Update', tag: 'button' }).click();
+  cy.reload();
+});
+
+Then(
+  "the viewer user is listed as a viewer in the dashboard's share list",
+  () => {
+    cy.getByLabel({ label: 'share', tag: 'button' }).click();
+    cy.get('*[class^="MuiList-root"]')
+      .eq(1)
+      .children()
+      .contains(dashboardViewerUser.login)
+      .should('exist');
+    cy.getByTestId({ testId: 'role-input' })
+      .eq(0)
+      .contains('viewer')
+      .should('exist');
+    cy.get('[data-state="added"]').should('not.exist');
+    cy.getByLabel({ label: 'Cancel', tag: 'button' }).click();
+    cy.logout();
+    cy.getByLabel({ label: 'Alias', tag: 'input' }).should('exist');
+  }
+);
+
+When('the viewer user logs in on the platform', () => {
+  cy.loginByTypeOfUser({
+    jsonName: dashboardViewerUser.login,
+    loginViaApi: false
+  });
+  cy.visit(`${Cypress.config().baseUrl}/centreon/home/dashboards`);
```

Comment:
not needed to specify base url when you navigate internally
```suggestion
  cy.visit('/centreon/home/dashboards');
```